react/__tests__/__snapshots__/Footer.test.js.snap

```diff
@@ -0,0 +1,398 @@
+                              href="mockedUrl"
+                            >
+                              Some title
+                            </a>
+                          </div>
+                        </div>
+                      </div>
+                    </div>
+                  </div>
+                </div>
+              </div>
+            </div>
+            <div
+              class="socialNetworkContainer pv5-s pa1-ns"
+            >
+              <div
+                class="listContainer pl0 w-100 mt0-ns listContainerRightAligned ml-auto-m listContainerHorizontal w-auto-m"
+              >
+                <span
+                  class="listTitle t-small ma0 db"
+                >
+                  Social Networks
+                </span>
+                <ul
+                  class="list list flex flex-column flex-wrap pa0 mb0 mt3 listHorizontal flex-row flex-wrap"
+                >
+                  <li
+                    class="listItem mr0 mr3 listItemHorizontal"
+                  />
+                </ul>
+              </div>
+            </div>
+          </nav>
+          <div
+            class="container payment pv5-s flex justify-between bg-base c-muted-1"
+          >
+            <div
+              class="paymentMatrix flex flex-wrap"
+            >
+              <div
+                class="paymentMatrixItem"
+              >
+                <div
+                  class="listContainer pl0 w-100 mt0-ns listContainerHorizontal w-auto-m"
+                >
+                  <span
+                    class="listTitle t-small ma0 db ttu"
+                  >
+                    Title payment
+                  </span>
+                  <ul
+                    class="list list flex flex-column flex-wrap pa0 mb0 mt3 listHorizontal flex-row flex-wrap"
+                  >
+                    <li
+                      class="listItem mr0 mr3 listItemHorizontal"
+                    />
+                  </ul>
+                </div>
+              </div>
+            </div>
+          </div>
+          <div
+            class="container informationContainer pt5-s flex justify-between bg-base c-muted-1"
+          >
+            <div
+              class="textContainer w-100-s pb5-s w-80-ns flex flex-wrap"
+            />
+            <div
+              class="listContainer pl0 w-100 mt0-ns"
+            >
+              <span
+                class="listTitle t-small ma0 db ttu"
+              />
+              <ul
+                class="list list flex flex-column flex-wrap pa0 mb0 mt3"
+              >
+                <li
+                  class="listItem mr0 mr3"
+                >
+                  <img
+                    src="imageURL"
+                  />
+                </li>
+              </ul>
+            </div>
+          </div>
+        </div>
+      </section>
+    </footer>
+  </div>,
+  "debug": [Function],
```

Comment:
These functions are related to the test framework, I believe asFragment should be used to generate the snapshot.